data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,646 @@ mission "Remnant: Learn Sign Follow Up"
-			choice
-				`	"I'd be happy to."`
-				`	"I'd prefer not to."`
+			
+			label terminus1
+			`	While wandering the spaceport, you think back to the scientsits researching the wormhole in Terminus. Given that the Remnant value their secrecy, they may be interested in knowing about this.`
+			choice
+				`	(Find a prefect to tell.)`
+				`	(Don't tell the Remnant.)`
 					decline
-			`	"Excellent. See what you can find out about their research and results, but it would probably be wise to avoid making direct contact with them."`
+			
+			label telling
+			apply
+				set "remnant: told about deep"
+			`	You ask one of the Remnant in the spaceport where you would be able to find a prefect and they point you to what looks like an information desk of sorts. There you tell the attendant that you have something potentially important to discuss in regards to Remnant security. They tap a few buttons then give you directions down a hallway. You quickly find yourself in an armory where several Remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`
+			`	"Greetings, Captain. I am Prefect Torza. I was told you had information that could be important to our security." He takes a seat on a bench on the far side of the room from the other Remnant and gestures for you to do the same. "What do you have to tell us?"`
+			branch terminus2
+				not "Deep: Remnant Research: done"
+			
+			choice
+				`	"I helped a group of scientists from the Deep to discover the Ember Waste."`
+				`	"A group of scientists from the Deep has discovered the Ember Waste."`
+					goto discovered
+			
+			apply
+				set "remnant: admitted to helping deep"
+			`	Torza's eyes go wide. "Why would you do this?" he asks. "Do you not understand that our secrecy is of upmost importance?"`
+			choice
+				`	"I don't believe that they will be a threat to you."`
+					goto threat
+				`	"I'm sorry, but I helped them before coming to understand this."`
+			
+			`	Torza seems to relax slightly. "This is unfortunate. I can not get angry at you for that which you did not understand. But now that you do understand, seek to not make this same mistake in the future.`
+			`	"What do they know?"`
+				goto knowledge
+			
+			label terminus2
+			choice
+				`	"I helped a group of scientists who were researching the wormhole in Terminus."`
+				`	"I discovered that a group of scientists are researching the wormhole in Terminus."`
+			
+			`	"Do they know how to enter it?" Torza asks, not sounding concerned with this new information.`
+			`	"No," you say. "But they know, or at least think, that it is a wormhole, and are attempting to learn how to enter it."`
+			`	"Well it is unlikely that they will ever learn. The only way to stabilize the wormholes is by using a Key Stone, but I have never heard of one being found in Ancestral space. Learning what these scientists know would still be helpful, though. Would you be able to get us a copy of their findings?"`
```

Comment:
Given that providing a batch of keystones from the Hai is a main mission that the player has likely done at this point, it seems odd that Torza would be unaware of a source of crystals outside the waste. His statement is technically true about Ancestral space, but seems kind of naive nonetheless.

Reply:
Should I add mention of Hai space from Torza or leave it as is?